rsconf/package_data/sirepo_jupyterhub/conf.py.jinja

```diff
@@ -26,7 +26,8 @@ c.DockerSpawner.image = '{{ sirepo_jupyterhub.jupyter_docker_image }}'
 c.DockerSpawner.image_whitelist = []
 c.DockerSpawner.network_name = 'host'
 c.DockerSpawner.use_internal_ip = True
-c.JupyterHub.authenticator_class = sirepo.jupyterhub.Authenticator
+c.JupyterHub.authenticator_class = getattr(sirepo.jupyterhub, '{{ sirepo_jupyterhub.authenticator_class }}')
+c.SirepoAuthenticator.sirepo_uri = '{{ sirepo_jupyterhub.sirepo_uri }}'
```

Comment:
I don't think this will work, because it may not be defined so better to use jinja:

```txt
{% if 'moderated_sim_types' in sirepo_jupyterhub %}
c.JupyterHub.authenticator_class = sirepo.jupyterhub.SirepoAuthenticator
c.SirepoAuthenticator.sirepo_user = '{{ sirepo_jupyterhub.sirepo_uri }}'
{% else %}
{%#TODO remove #%}
c.JupyterHub.authenticator_class = sirepo.jupyterhub.Authenticator
{% endif %}
```

Reply:
87e5cf5